spec/haskell/src/SEL4/API/Syscall.lhs

```diff
@@ -121,9 +121,9 @@ System call events are dispatched here to the appropriate system call handlers,
 Interrupt handling is performed by "handleInterrupt", defined in \autoref{sec:object.interrupt.kernel.handling}.
 
 > handleEvent Interrupt = withoutPreemption $ do
->     active <- doMachineOp (getActiveIRQ False)
 >     updateTimeStamp
 >     checkBudget
+>     active <- doMachineOp (getActiveIRQ False)
```

Comment:
what changed? was there an error in the spec, or was something else involved in order to require this change?